lib/network/http2_network.go

```diff
@@ -10,25 +10,33 @@ import (
 	"boscoin.io/sebak/lib/common"
 	"github.com/gorilla/handlers"
 
+	"github.com/gorilla/mux"
 	"golang.org/x/net/http2"
 )
 
 type Handlers map[string]func(http.ResponseWriter, *http.Request)
 
+const (
+	UrlNodePrefix = "/node"
+	UrlApiPrefix  = "/api"
+)
+
```

Comment:
In the below code for `HTTP2Network.routers`, you can define url path prefix and router name too.
```go
const (
	RouterNameNode = "node"
	RouterNameAPI  = "node"
)

var (
	UrlPathPrefixNode = fmt.Sprintf("/%s", RouterNameNode)
	UrlPathPrefixAPI  = fmt.Sprintf("/%s", RouterNameAPI)
)
```